utils/kubernetes/kube_state_processor.py

```diff
@@ -20,13 +25,38 @@ def __init__(self, kubernetes_check):
             # 'unknown':   AgentCheck.UNKNOWN
         }
 
+        # these metrics will be extracted with all their labels and reported as-is with their corresponding metric name
+        self.metric_to_gauge = {
+            # message.metric: datadog metric name
+            'kube_node_status_capacity_cpu_cores': NAMESPACE + '.node.cpu_capacity',
+            'kube_node_status_capacity_memory_bytes': NAMESPACE + '.node.memory_capacity',
+            'kube_node_status_capacity_pods': NAMESPACE + '.node.pods_capacity',
+            'kube_node_status_allocatable_cpu_cores': NAMESPACE + '.node.cpu_allocatable',
+            'kube_node_status_allocatable_memory_bytes': NAMESPACE + '.node.memory_allocatable',
+            'kube_node_status_allocatable_pods': NAMESPACE + '.node.pods_allocatable',
+            'kube_deployment_status_replicas_available': NAMESPACE + '.deployment.replicas_available',
+            'kube_deployment_status_replicas_unavailable': NAMESPACE + '.deployment.replicas_unavailable',
+            'kube_deployment_status_replicas_updated': NAMESPACE + '.deployment.replicas_updated',
+            'kube_deployment_spec_replicas': NAMESPACE + '.deployment.replicas_desired',
+            'kube_pod_container_resource_requests_cpu_cores': NAMESPACE + '.container.cpu_requested',
```

Comment:
I think what makes the CI fail is that we need to merge the `protobuf.bin` fixture, otherwise the metric doesn't exist. Tests pass locally when I copy the files.